### PR TITLE
Optimize `<img />` loading 🖼️ 

### PR DESCRIPTION
**Pull request by sanjaiyan-dev**
Optimize `` loading 🖼️ 

- [Lazy loading the image](https://web.dev/lazy-loading-images/#images-inline-browser-level)
- [Adding `decoding` attr to `` element.](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/decoding)

### CODE REVIEW

**Comment by sanjaiyan-dev** [comment]
> Optimize `` loading 🖼️ 
> 
> - [Lazy loading the image](https://web.dev/lazy-loading-images/#images-inline-browser-level)
> - [Adding `decoding` attr to `` element.](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/decoding)

For more information about lazy loading -:
https://web.dev/browser-level-image-lazy-loading

**Comment by luisorbaiceta** [comment]
Does not affect large contentful paint so I agree with parallelizing and lazy loading the assets. It does affect cumulative layout shift  in the organizations page though. Could you fix that? The height property will let the browser know beforehand how much space has to save to avoid this issue

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by jsumners** [comment]
> The height property will let the browser know beforehand how much space has to save to avoid this issue

Yes please. I really dislike it when a site shifts around because image heights were not specified.

**Comment by sanjaiyan-dev** [comment]
Hope I did it correctly :)

File: src/website/layouts/organisations.html [hunk 1]
```diff
@@ -21,7 +21,7 @@ <h1 class="title">
 					{% for organization in data.organisations | shuffle %}
 						<li>
 							<a href="{{ organization.link }}" target="_blank" rel="noopener" rel="nofollow">
-								<img src="/{{ hashes['images/organisations/' + organization.image] }}" alt="{{ organization.name }} is using Fastify"/>
+								<img height="60" src="/{{ hashes['images/organisations/' + organization.image] }}" alt="{{ organization.name }} is using Fastify" loading="lazy" decoding="async" />
```
Comment (Uzlopak) [inline src/website/layouts/organisations.html]: isnt this set by css?

Comment (luisorbaiceta) [reply src/website/layouts/organisations.html]: https://web.dev/optimize-cls/

Comment (luisorbaiceta) [reply src/website/layouts/organisations.html:24]: `width` needs to be set too to fully prevent CLS. Lets see if with `160` we get a good generic aspect ratio

**Review by luisorbaiceta — CHANGES_REQUESTED** [review]

File: src/website/layouts/organisations.html (hunk 1 above)
Comment (luisorbaiceta) [inline src/website/layouts/organisations.html]: `width` needs to be set too, to fully prevent CLS. Lets see if with `160` we get a good generic aspect ratio for the logos

**Review by luisorbaiceta — CHANGES_REQUESTED** [review]

File: src/website/layouts/organisations.html (hunk 1 above)
Comment (luisorbaiceta) [reply src/website/layouts/organisations.html]: Digging further I've found that either we set a container with a fixed aspect ratio and make the logos fill or we target the image logos like this:

```css
.logo-img {
  aspect-ratio: attr(width) / attr(height);
}
```

Comment (sanjaiyan-dev) [reply src/website/layouts/organisations.html]: > `width` needs to be set too, to fully prevent CLS. Lets see if with `160` we get a good generic aspect ratio for the logos

Done 🤝

**Review by luisorbaiceta — APPROVED** [review]
👌

**Pull request merged**